workers/dockerfiles/Dockerfile.salmon

```diff
@@ -23,41 +30,29 @@ WORKDIR /home/user
 
 # Install Salmon
 ENV SALMON_VERSION 0.9.1
-
 RUN wget https://github.com/COMBINE-lab/salmon/releases/download/v${SALMON_VERSION}/Salmon-${SALMON_VERSION}_linux_x86_64.tar.gz
-
 RUN tar -xzf Salmon-${SALMON_VERSION}_linux_x86_64.tar.gz
-
-# Salmon can extract to a different directory than the name of the tar file.
-RUN cp `tar -tzf Salmon-${SALMON_VERSION}_linux_x86_64.tar.gz | head -1 | cut -f1 -d"/"`/bin/salmon /usr/local/bin
-
-RUN cp `tar -tzf Salmon-${SALMON_VERSION}_linux_x86_64.tar.gz | head -1 | cut -f1 -d"/"`/lib/* /usr/local/lib
-
-RUN rm -r Salmon*
+RUN rm -f Salmon-${SALMON_VERSION}_linux_x86_64.tar.gz
 # End Salmon installation.
 
-# Package list based on:
```

Comment:
I removed these packages because these packages were required to compile salmon, and we are using pre-compiled version. (As a matter of fact, most of these packages have already been installed implicitly because they were the dependencies of `r-base`.)